### PR TITLE
Modified the Login UI and Tried Implementing Tests for the Search Bar

### PR DESCRIPTION
**Pull request by imaneb942**
We have encountered several issues with the RangeSlider, in fact any compose that takes a RangeSlider breaks when tested, the issue comes from JetPack Compose software, we will probaby change the Range Slider to simple slider, althought visually it won't be the same but it will be more robust, so after a long time of testing we haven't achieved anything interesting.
We also changed the Login Screen UI to match the Figma desired rendering.
You will also find a draft of a Chips Model that we want to implement for our tag filtering next time. (commented for obvious reasons)

### CODE REVIEW

File: app/src/main/java/com/github/wanderwise_inc/app/ui/home/SearchBar.kt [hunk 1]
```diff
@@ -49,55 +49,70 @@ fun SearchBar(
       leadingIcon = {
         Icon(
             painter = painterResource(id = R.drawable.les_controles),
-            contentDescription = null,
+            contentDescription = "les_controles",
```
Comment (engjellismaili) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/home/SearchBar.kt:52]: name in french ?

File: app/src/main/java/com/github/wanderwise_inc/app/ui/signin/LoginScreen.kt [hunk 2]
```diff
@@ -33,43 +35,59 @@ import com.github.wanderwise_inc.app.data.GoogleSignInLauncher
 fun LoginScreen(googleSignInLauncher: GoogleSignInLauncher, modifier: Modifier = Modifier) {
   Box(modifier = modifier.requiredWidth(width = 1280.dp).requiredHeight(height = 1100.dp)) {
     Image(
-        painter = painterResource(id = R.drawable.underground_2725336_1280),
-        contentDescription = "underground-2725336_1280 1",
+        painter = painterResource(id = R.drawable.logo_swent),
+        contentDescription = "new_logo_swent 1",
```
Comment (ThomasPFMuller) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/signin/LoginScreen.kt:39]: Nice change, the logo is definitely cooler 😎

File: app/src/main/java/com/github/wanderwise_inc/app/ui/signin/LoginScreen.kt
Comment (engjellismaili) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/signin/LoginScreen.kt:1]: Lgtm good modification and clean screen

File: app/src/test/java/com/github/wanderwise_inc/app/ui/signin/LoginScreenKtTest.kt
Comment (engjellismaili) [inline app/src/test/java/com/github/wanderwise_inc/app/ui/signin/LoginScreenKtTest.kt:1]: lgtm

**Review by ThomasPFMuller — APPROVED** [review]
Nice changes overall, the removed classes were unused so removing them increases code coverage. Removing the test is understandable, we'll work on making one that actually works 😄

**Review by engjellismaili — COMMENTED** [review]
Overall good code and great testing, just a thing to modifiy. Well done

File: app/src/main/java/com/github/wanderwise_inc/app/ui/home/SearchBar.kt (hunk 1 above)
Comment (imaneb942) [reply app/src/main/java/com/github/wanderwise_inc/app/ui/home/SearchBar.kt]: modified thanks

File: app/src/main/java/com/github/wanderwise_inc/app/ui/home/SearchBar.kt [hunk 3]
```diff
@@ -49,55 +49,70 @@ fun SearchBar(
...
       singleLine = true,
       shape = RoundedCornerShape(30.dp),
       modifier =
           Modifier.background(MaterialTheme.colorScheme.primaryContainer)
               .fillMaxWidth()
-              .padding(5.dp)
-              .testTag("SearchBar"))
+              .padding(5.dp))
+  DropdownSearch(isDropdownOpen, sliderPositionPriceState, sliderPositionTimeState)
+}
```
[the remark this answers is not in the document]
Comment (imaneb942) [reply app/src/main/java/com/github/wanderwise_inc/app/ui/home/SearchBar.kt]: thanks Thomas!

**Pull request merged**